.github/workflows/validate.yml

```diff
@@ -0,0 +1,14 @@
+name: Validate JSONs
+
+on: [push, pull_request]
+
+jobs:
+  verify-json-validation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Validate package.json against local schema
+        uses: cardinalby/schema-validator-action@v3
+        with:
+            file: 'verified-mods.json'
+            schema: '.github/schema.json'
```

Comment:
Insert trailing newline? 👉👈

Reply:
Addressed in 51c12c32ed20f85bfe55da93ee12d16215155f07